src/app/components/elements/panels/UserDetails.tsx

```diff
@@ -141,6 +148,31 @@ export const UserDetails = (props: UserDetailsProps) => {
                 </div>
             </Col>}
         </Row>
+        {SITE_SUBJECT === SITE.CS && allowProgrammingLanguageOption && <Row className="mt-3">
+            <Col md={6}>
+                <FormGroup>
+                    <Label className="d-inline-block pr-2 form-required" htmlFor="programming-language-select">
```

Comment:
This has the required field style but the form does not require it 

```suggestion
                    <Label className="d-inline-block pr-2" htmlFor="programming-language-select">
```